File: exercises/D110/solution.pas
var
    c1, r1, c2, r2: char;
    c, r: longint;
begin
    readln(c1, r1);
    readln(c2, r2);
    c := abs(ord(c1) - ord(c2));
    r := abs(ord(r1) - ord(r2));
    if (c > r) then
      writeln(c)
    else
      writeln(r)
end.

File: exercises/D110/solution.cpp
#include <iostream>
using namespace std;
int main() {
    char c1, r1, c2, r2;
    cin >> c1 >> r1;
    cin >> c2 >> r2;
    int c = abs(c1 - c2);
    int r = abs(r1 - r2);
    if (c > r) {
        cout << c << endl;
    } else  {
        cout << r << endl;
    }
}
